beast/examples/phat_small/datamodel.py

```diff
@@ -145,10 +145,10 @@
 #      (choices: parsec12s_r14, parsec12s, 2010, 2008, 2002)
 #   MISTWeb() -- `rotation` param (choices: vvcrit0.0=default, vvcrit0.4)
 #
-# Default: PARSEC+CALIBRI
-#oiso = isochrone.PadovaWeb()
+# Default: PARSEC+COLIBRI
+oiso = isochrone.PadovaWeb()
 # Alternative: PARSEC1.2S -- old grid parameters
-oiso = isochrone.PadovaWeb(modeltype='parsec12s', filterPMS=True)
+#oiso = isochrone.PadovaWeb(modeltype='parsec12s', filterPMS=True)
```

Comment:
Again very careful, some parameters do not produce correct files on the parsec website when you use this model version.

Reply:
Can you be more specific?
I've run the BEAST with the parsec12s option and it works.  Maybe the BEAST is not sensitive to the differences?